### ISSUE TITLE
Force remove packages after unsuccesful remove

### ISSUE THREAD

**Issue by D3VL-Jack** [fpv-wtf/wtfos-configurator#179]
Yesterday I had a package (of my own making) fail to be removed using the configurator, the configurator could check after removing if it's really removed, if it hasn't been then offer the user to force remove it.

Alternatively, make all package removals use the `--force-remove` flag

https://user-images.githubusercontent.com/21148384/193474557-ee3ea51c-44c4-4085-b475-503d438e012b.mp4

![cmd_qNtcouW073](https://user-images.githubusercontent.com/21148384/193474565-a7586483-dc45-4220-9c72-3ba092c7d1a8.png)

**Dear reader: this was a beta package, this is not behavior of release versions of `fcc-unlock`**

**Comment by stylesuxx** [comment]
Sure, I can force remove the packages - can't see what the downside of this would be anyway.
Generally removal and installation could use some improvement in error handling.

What was the underlying issue? prerm not being executable, I assume?

**Comment by D3VL-Jack** [comment]
Awesome.
Yep was, missing execute permissions on prerm

**Comment by j005u** [comment]
I would definitely not default to --force remove. Checking the exit code of prerm is there for a reason. Just ignoring issues could leave users in a bad state.

**Comment by stylesuxx** [comment]
Yeah, true, if more complex stuff - apart from disabling a service - is happening in the prerm, that would be bad.
